packages/browsers/src/browser-data/chromium.ts

```diff
@@ -30,6 +31,7 @@ function folder(platform: BrowserPlatform): string {
       return 'Linux_x64';
     case BrowserPlatform.MAC_ARM:
       return 'Mac_Arm';
+    case BrowserPlatform.LINUX_ARM:
```

Comment:
@OrKoN Are we sure that `LINUX_ARM` should return `Mac` here?

Reply:
Looks like a typo but it should be handled correctly by line 15